docs/scripts/buildApi.ts

```diff
@@ -55,6 +55,8 @@ interface DescribeablePropDescriptor {
   type: PropTypeDescriptor;
 }
 
+const cssComponents = ['Box', 'Grid', 'Typography'];
```

Comment:
Box has no API page for some reason.

Reply:
I noticed but decided to leave the component here, in case we migrate it in the future..